google_cloud_vision_msgs/Vertex[] vertices
